src/client/src/apps/MainRoute/widgets/spotTile/epics/rfqEpics.ts

```diff
@@ -87,7 +96,7 @@ export const rfqRequestEpic: ApplicationEpic<{}> = (action$, state$) =>
       return rfqService(
         action.payload,
         state$.value.currencyPairs,
-        state$.value.spotTilesData,
```

Comment:
types were updated to represent the case where we might not get spotTilesData for certain currencies. So now we need to handle unhappy path with filter